.github/workflows/tests.yml

```diff
@@ -259,6 +259,7 @@ jobs:
           -e TEST_GROUP
           ghcr.io/conda/conda-ci:main-linux-python${{ matrix.python-version }}${{ matrix.default-channel == 'conda-forge' && '-conda-forge' || '' }}
           bash -c "source /opt/conda/etc/profile.d/conda.sh; \
+                   sudo /opt/conda/bin/conda install --quiet -y -c defaults --file tests/requirements.txt; \
                    pytest --cov=conda -k test_DepsModifier_contract"
```

Comment:
This could use `unit.sh`, which could take some CLI args for pytest (default "-k "not integration").